app/src/main/java/de/danoeh/antennapod/fragment/ItemFragment.java

```diff
@@ -29,6 +31,15 @@
 import com.skydoves.balloon.ArrowOrientation;
 import com.skydoves.balloon.Balloon;
 import com.skydoves.balloon.BalloonAnimation;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.greenrobot.eventbus.EventBus;
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
+
+import java.util.List;
+import java.util.Locale;
+
```

Comment:
Please don't change code in this unrelated file to keep the git history clean

Reply:
I try not to, but the tests wouldn't pass without it. There's probably a way to avoid, but I don't know exactly and didn't think it would be a problem if it just involved checkstyle fixes.

Reply:
If a file is exactly the same as the current `develop` version (e.g. no diff displayed on GitHub), Checkstyle shouldn't complain about that file (even if there are checkstyle violations). Could you please try making the file exactly the same?

Reply:
I think, in this case, we should keep the change since it was change to fix checkstyle errors. It would be a bigger hassle have to revert it and then go back to fix errors your going to have to fix anyways.

Reply:
The problem is that unrelated changes like this also often cause merge conflicts. I just reverted them - let's see if CI passes. By the way, reverting is pretty fast if you simply do `git checkout develop <filename>`.

Reply:
Tests nearly passed :tada: Just some checkstyle errors in the newly created file.